### DISCUSSION TITLE
Functions

### DISCUSSION THREAD

**Discussion by JayMartin05** [JustinRoberts5760/convoy-sGameOfLife#13] [Ideas]
What functions do we think will be needed?

**Comment by JustinRoberts5760** [comment]
//Create a way to accept user input

//Create a function where each cell with one or no neighbors dies,
//as if by solitude.

//Create a function where each cell with four or more neighbors dies,
//as if by overpopulation.

//Create a function where each cell with two or three neighbors survives.

//Create a function where (For a space that is empty or unpopulated)
//each cell with three neighbors becomes populated.
